lib/sinon/util/fake_server.js

```diff
@@ -80,12 +81,14 @@ var fakeServer = {
 
         return server;
     },
+
     configure: function (config) {
         var whitelist = {
```

Comment:
I was a bit confused by this whitelist, although I know it was there earlier. The values of this object is not used at all, so I assume that the only reason to have this as an object, instead of a list (array) of properties is to make the `hasOwnProperty` symmetrical for both the `whitelist` and the `config` object of purely visual reasons. Am I right in thinking so?